python/ml_wrappers/model/image_model_wrapper.py

```diff
@@ -14,6 +14,9 @@
 from ml_wrappers.model.pytorch_wrapper import WrappedPytorchModel
 from ml_wrappers.model.wrapped_classification_model import \
     WrappedClassificationModel
+import torch
+import torchvision 
```

Comment:
Curious if `torchvision` is in the dependencies and/or needs to be added?

Reply:
it is not in the dependency file. @imatiach-msft would you recommend adding to dependencies?

Reply:
Actually, I forgot, we shouldn't depend on it in ml-wrappers anyway since users should be able to use this package without pytorch.  So we definitely shouldn't have it in the setup.py.  We also wrap all imports in the ml-wrappers package from other frameworks like pytorch, tensorflow, xgboost etc in try/catch since the imports should not cause ml_wrappers package as a whole to fail.

Reply:
Got it - so to summarize, torch/torchvision etc should not be added to requirements.txt, but can still be imported in individual files w/in ml-wrappers